src/Microsoft.DocAsCode.Plugins/IHrefInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Plugins
+{
+    public interface IHrefInfo
+    {
+        string FileInSource { get; }
+        string FileInDest { get; }
+        string TargetFileInSource { get; }
```

Comment:
What is FileInSouce and TargetFileInSource... why Source contains Target File.... Is any file in source SourceFile?

---
In reply to: [95518402](https://github.com/dotnet/docfx/pull/1146#discussion_r95518402) [](ancestors = 95518402)